### ISSUE TITLE
docs: make multiple versions available from a drop down in the header

### ISSUE THREAD

**Issue by krancour** [akuity/kargo#3171]
This was briefly discussed the other day.

Each time we have a minor release, that release's branch becomes the branch that the main docs are served from. Older release branches move to subdomains like release-1-1.docs.kargo.io

We need to add a drop down to the header so that docs for older releases can be selected. It just needs to redirect readers to the correct subdomain.

I think @rpelczar mentioned that Docusaurus may actually have some built-in feature to help with this, although I've not explored that.

**Comment by rpelczar** [comment]
Yes, it has a built-in feature: https://docusaurus.io/docs/versioning
